boards/arm/stm32h7/nucleo-h743zi/scripts/flash.ld

```diff
@@ -172,9 +172,12 @@ SECTIONS
 
     /* Emit the the D3 power domain section for locating BDMA data */
 
-    .sram4 :
-    {
-    } > sram4
+	.sram4_reserve :
```

Comment:
```suggestion
	.sram4_reserve (NOLOAD) :
```
A OK Once enabled 
38000800 A _sram4_reserve_end
38000000 A _sram4_reserve_start
38000400 00000400 d g_spi6_rxbuf
38000000 00000400 d g_spi6_txbuf

Except:

-rwxrwxr-x  1 david_s5 david_s5 805308416 Mar 27 05:34 nuttx.bin

Yikes!

With **NOLOAD**

108 -rwxrwxr-x  1 david_s5 david_s5  108708 Mar 27 05:39 nuttx.bin

24000074 00000048 d g_spi6dev
38000400 00000400 b g_spi6_rxbuf
38000000 00000400 b g_spi6_txbuf
0800207e 0000001a T stm32_spi6select
08002098 00000018 T stm32_spi6status
38000800 A _sram4_reserve_end
38000000 A _sram4_reserve_start

Reply:
Ouch! Thanks for reviewing. I'm glad you caught that!